docs/errors-to-ignore.md

```diff
@@ -0,0 +1,31 @@
+## Errors To Ignore
+
+Some parts of Laravel are currently too magical for Larastan/PHPStan to understand.
+
+If you hit those errors in your project, you can add them to your `phpstan.neon` as needed.
+For example:
+
+```neon
+ignoreErrors:
+- '#Result of function abort \(void\) is used#'
+```
+
+> Tip: If you set `reportUnmatchedIgnoredErrors: true`, you will be notified once Larastan
```

Comment:
This defaults to `true` in PHPStan. So maybe this tip is not needed.

Reply:
Replaced with a link to PHPStan's docs.